ansible_runner/interface.py

```diff
@@ -90,6 +90,7 @@ def run(**kwargs):
       - Path to the inventory file in the ``private_data_dir``
       - Native python dict supporting the YAML/json inventory structure
       - A text INI formatted string
+      - A list of inventory sources, or an empty list to disable passing inventory
```

Comment:
I tried to read a lot of the docs to make sense of this.... so I think the bottom line is that this removes extra processing in the CLI interface with the result being that what you pass the the ansible-runner CLI as `--inventory` becomes the actual thing passed to Ansible via `-i`. There's also the additional detail of allowing use of lists, but this can't quite be utilized via the runner CLI, which is certainly fine.

Reply:
You're right, most of the extra focus is on module interaction with the thought that *that* is usually the point where you want *more* flexibility beyond the `private_data_dir`.

In the future we can expand the functionality of the cli if that's needed.

Reply:
If you pass `None` to inventory does it also disable passing inventory?